src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its engine and began to work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck turned off its engine and stopped working.");
```

Comment:
Similarly, the message 'Truck turned off its engine and stopped working.' could be more informative. It's recommended to rephrase it to clearly indicate the type of Machine, for example: 'Truck has stopped working.'